terminus-ui/scss/helpers/test-sass.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef */
```

Comment:
What is the undefined item we are ignoring?

Reply:
The True mixins: `describe` and `test`. I can probably make two one-liners, but it was useful having the whole doc when in discovery mode.

Reply:
Hmm I wonder why this is needed but isn't needed in all the other files that use those two..

Reply:
Alright, figured out why this was happening. Our ESLint configuration sets the `jest` environment for files ending with `spec` or `mock` so it wasn't catching this new file naming convention. We can remove this ESLint comment by extending the overrides section of our ESLint configuration:

### `.eslintrc.js`

```javascript
module.exports = {
  "extends": ["@terminus/eslint-config-frontend/development"],
  "parserOptions": {
    "ecmaVersion": 6,
    "project": "./tsconfig.json",
    "sourceType": "module"
  },
  "rules": {
    "no-console": [
      "error",
      {
        "allow": [
          "warn"
        ]
      }
    ]
  },
  'overrides': [
    {
      'files': [
        '**/*.spec.ts',
        '**/*.mock.ts',
        '**/test-sass.js',
      ],
      'env': {
        'jest': true,
      },
    },
  ],
}
```

### `.eslintrc.ci.js`

```javascript
module.exports = {
  "extends": ["@terminus/eslint-config-frontend"],
  "parserOptions": {
    "ecmaVersion": 6,
    "project": "./tsconfig.json",
    "sourceType": "module"
  },
  "rules": {
    "no-console": [
      "error",
      {
        "allow": [
          "warn"
        ]
      }
    ]
  },
  'overrides': [
    {
      'files': [
        '**/*.spec.ts',
        '**/*.mock.ts',
        '**/test-sass.js',
      ],
      'env': {
        'jest': true,
      },
    },
  ],
}
```